databricks-sdk-java/src/main/java/com/databricks/sdk/core/DatabricksConfig.java

```diff
@@ -83,9 +83,6 @@ public class DatabricksConfig {
   @ConfigAttribute(env = "ARM_ENVIRONMENT")
   private String azureEnvironment;
 
-  @ConfigAttribute(env = "DATABRICKS_AZURE_LOGIN_APP_ID", auth = "azure")
```

Comment:
This is now derived from the DatabricksEnvironment.

Reply:
Does this impact existing users which are using the env var?

Reply:
FYI: this is only dev/staging users. All prod users use the same login ID. It shouldn't matter because there is only one correct login app ID to use for your workspace, and it is coupled to the environment you're in.